roles/youtubedl_material/defaults/main.yml

```diff
@@ -0,0 +1,157 @@
+youtubedl_material_web_domain: "{{ user.domain }}"
+youtubedl_material_web_port: "17442"
+youtubedl_material_web_url: "{{ 'https://' + youtubedl_material_web_subdomain + '.' + youtubedl_material_web_domain
+                     if (youtubedl_material_web_subdomain | length > 0)
+                     else youtubedl_material_web_domain }}"
+
+################################
+# DNS
+################################
+
+youtubedl_material_dns_record: "{{ youtubedl_material_web_subdomain }}"
+youtubedl_material_dns_zone: "{{ youtubedl_material_web_domain }}"
+youtubedl_material_dns_proxy: "{{ dns.proxied }}"
+
+################################
+# Traefik
+################################
+
+youtubedl_material_traefik_sso_middleware: "{{ traefik_default_sso_middleware }}"
+youtubedl_material_traefik_middleware: "{{ traefik_default_middleware + ',' + youtubedl_material_traefik_sso_middleware
+                                if (youtubedl_material_traefik_sso_middleware | length > 0)
+                                else traefik_default_middleware }}"
+youtubedl_material_traefik_certresolver: "{{ traefik_default_certresolver }}"
+youtubedl_material_traefik_enabled: true
+
+################################
+# Docker
+################################
+
+# Container
+youtubedl_material_docker_container: "{{ youtubedl_material_name }}"
+
+# Image
+youtubedl_material_docker_image_pull: true
+youtubedl_material_docker_image_tag: "latest"
+youtubedl_material_docker_image: "tzahi12345/youtubedl-material:{{ youtubedl_material_docker_image_tag }}"
+
+# Ports
+youtubedl_material_docker_ports_defaults: []
+youtubedl_material_docker_ports_custom: []
+youtubedl_material_docker_ports: "{{ youtubedl_material_docker_ports_defaults
+                             + youtubedl_material_docker_ports_custom }}"
+
+# Envs
+youtubedl_material_docker_envs_default:
+  PUID: "{{ uid }}"
+  PGID: "{{ gid }}"
+  TZ: "{{ tz }}"
+  # ytdl_use_local_db: 'false'
+  write_ytdl_config: 'true'
+youtubedl_material_docker_envs_custom: {}
+youtubedl_material_docker_envs: "{{ youtubedl_material_docker_envs_default
+                            | combine(youtubedl_material_docker_envs_custom) }}"
+
+# Commands
+youtubedl_material_docker_commands_default: []
+youtubedl_material_docker_commands_custom: []
+youtubedl_material_docker_commands: "{{ youtubedl_material_docker_commands_default
+                                + youtubedl_material_docker_commands_custom }}"
+
+# Volumes
+youtubedl_material_docker_volumes_default:
+  - "{{ youtubedl_material_paths_location }}/appdata:/app/appdata"
+  - "{{ youtubedl_material_paths_location }}/users:/app/users"
+  - "{{ youtubedl_material_download_folder }}/audio:/app/audio"
+  - "{{ youtubedl_material_download_folder }}/video:/app/video"
+  - "{{ youtubedl_material_download_folder }}/subscriptions:/app/subscriptions"
+youtubedl_material_docker_volumes_custom: []
+youtubedl_material_docker_volumes: "{{ youtubedl_material_docker_volumes_default
```

Comment:
Look at other roles for a reference on how to format this and remove docker_volumes_downloads_common as it is already included